lib/src/components/viewOptions/ViewOptions.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react';
```

Comment:
💚💚💚 Probably debatable, but I feel like the dir name should be capitalised to `ViewOptions/ViewOptions.js` - as it is the component name.

TBF I am happy with either, but maybe we should discuss in FE chat to get a standard. I'll create a card 🌄

Reply:
DIRs shouldn't be capitalised (IMO). I've given a detailed reason in the past about why it's ideal not to capitalised folders but I'll have it dig it out.